HLTrigger/Configuration/python/Phase2/trackerLocalReco_cff.py

```diff
@@ -0,0 +1,84 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Why does this python file have a name which does not start with ```hltPhase2``` as all the others? 
Is this a copy of the 'official' trackerLocalReco? Then it should be imported from there and only the actually changed parameters modified.

Reply:
I was worried about the offline local reco differing from the online, but for trackerLocalReco I reckon the chances of this happening are very low. I will rename it` hltPhase2TrackerLocalReco`, import from the offline and modify the parameters if needed.

Reply:
This is addressed now!